app/inputs/searchable_association_input.rb

```diff
@@ -16,23 +16,15 @@ def input(wrapper_options = nil)
 
     def add_default_options
       if search_only?
-        input_html_options.merge!(search_options)
+        input_html_options.merge!(searchable_scope_options(show_subtext: options[:show_subtext],
+                                                           search_scope: options[:search_scope],
+                                                           excluded_ids: options[:excluded_ids]))
       else
         input_html_options[:class].push(:selectpicker)
       end
       options[:include_blank] = 'None' if options[:include_blank].nil?
     end
 
-    def search_options
```

Comment:
removing this stuff and now sharing the code between this simple form input and the search dropdown